### PR TITLE
[201911][Monit] Fix the template file of dhcp_relay

### PR DESCRIPTION
**Pull request by yozhao101**
Signed-off-by: Yong Zhao <yozhao@microsoft.com>

#### Why I did it
This PR aims to fix the bug in Monit template file of `dhcp_relay` container. 

If Multi-VLAN were configured on device, multiple `dhcrelay` processes will be spawned in `dhcp_relay` container. Then there will be an entry for each `dhcrelay` process in Monit configuration file of `dhcp_relay` container.

Currently Monit template file of `dhcp_relay` container can not be rendered correctly to generate configuration file and will cause Monit can not start up.

#### How I did it
I removed the indentation in the template file and remove the `minus` sign.

#### How to verify it
I verify this on DuT `str-msn2700-20` which was installed 20191130.78 image.
          
          ################################################################################
          ## Monit configuration file for dhcp_relay container
          ## process list:
          ## dhcrelay
          ################################################################################
          check program dhcp_relay|dhcrelay_Vlan102 with path "/usr/bin/process_checker dhcp_relay /usr/sbin/dhcrelay -d -m discard -a %h:%p %P --name-alias-map-file /tmp/port-name-alias-map.txt -id Vlan102"
          if status != 0 for 5 times within 5 cycles then alert repeat every 1 cycles
          
          check program dhcp_relay|dhcrelay_Vlan101 with path "/usr/bin/process_checker dhcp_relay /usr/sbin/dhcrelay -d -m discard -a %h:%p %P --name-alias-map-file /tmp/port-name-alias-map.txt -id Vlan101"
          if status != 0 for 5 times within 5 cycles then alert repeat every 1 cycles

#### Which release branch to backport (provide reason below if selected)
Monit is still used to monitor the running status of critical processes in 201911 branch.

- [ ] 201811
- [x ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

File: dockers/docker-dhcp-relay/base_image_files/monit_dhcp_relay.j2 [hunk 1]
```diff
@@ -29,7 +29,8 @@
           {# Check the running status of each DHCP relay agent instance #}
 check program dhcp_relay|dhcrelay_{{ vlan_name }} with path "/usr/bin/process_checker dhcp_relay /usr/sbin/dhcrelay -d -m discard -a %h:%p %P --name-alias-map-file /tmp/port-name-alias-map.txt -id {{ vlan_name }}"
     if status != 0 for 5 times within 5 cycles then alert repeat every 1 cycles
-        {%- endif -%}
+
+{% endif -%}
```
Comment (qiluo-msft) [inline dockers/docker-dhcp-relay/base_image_files/monit_dhcp_relay.j2:33]: This fix mess the indentation level and is hard to read.
How about just remove all the indentation of before `{`? You could add blanks between `{` and key word in the middle. And remove all `-` after `{` and before `}`.

Comment (yozhao101) [reply dockers/docker-dhcp-relay/base_image_files/monit_dhcp_relay.j2:33]: Will update. Thanks.

Comment (yozhao101) [reply dockers/docker-dhcp-relay/base_image_files/monit_dhcp_relay.j2]: @qiluo-msft Can you please help me review again?

**Review by qiluo-msft — APPROVED** [review]

**Review by prsunny — APPROVED** [review]

**Review by abdosi — APPROVED** [review]

**Pull request merged**